bug:  [plugin externalize-deps] EINVAL: invalid argument, stat

## System
Windows 10
deno 1.26.1

## Reproduction
1. `deno run -A --unstable npm:create-vite-extra`
2. select deno-vue
3. cd into folder
4. run `deno task dev`

## Full logs:
```
Warning deno task is unstable and may drastically change in the future
Task dev deno run -A --unstable --node-modules-dir npm:vite
✘ [ERROR] [plugin externalize-deps] EINVAL: invalid argument, stat

    vite.config.mjs:2:16:
      2 │ import vue from 'npm:@vitejs/plugin-vue'
        ╵                 ~~~~~~~~~~~~~~~~~~~~~~~~

failed to load config from D:\deno-vite-test2\vite.config.mjs
error when starting dev server:
Error: Build failed with 1 error:
vite.config.mjs:2:16: ERROR: [plugin: externalize-deps] EINVAL: invalid argument, stat
    at failureErrorWithLog (file:///D:/deno-vite-test2/node_modules/.deno/esbuild@0.15.10/node_modules/esbuild/lib/main.js:1566:15)
    at file:///D:/deno-vite-test2/node_modules/.deno/esbuild@0.15.10/node_modules/esbuild/lib/main.js:1024:28
    at runOnEndCallbacks (file:///D:/deno-vite-test2/node_modules/.deno/esbuild@0.15.10/node_modules/esbuild/lib/main.js:1438:61)
    at buildResponseToResult (file:///D:/deno-vite-test2/node_modules/.deno/esbuild@0.15.10/node_modules/esbuild/lib/main.js:1022:7)
    at file:///D:/deno-vite-test2/node_modules/.deno/esbuild@0.15.10/node_modules/esbuild/lib/main.js:1134:14
    at responseCallbacks.<computed> (file:///D:/deno-vite-test2/node_modules/.deno/esbuild@0.15.10/node_modules/esbuild/lib/main.js:671:9)
    at handleIncomingPacket (file:///D:/deno-vite-test2/node_modules/.deno/esbuild@0.15.10/node_modules/esbuild/lib/main.js:726:9)
    at w.readFromStdout (file:///D:/deno-vite-test2/node_modules/.deno/esbuild@0.15.10/node_modules/esbuild/lib/main.js:647:7)
    at w.emit (https://deno.land/std@0.159.0/node/_events.mjs:379:28)
    at ot (https://deno.land/std@0.159.0/node/_stream.mjs:10:32715)
```

## Additional Info
Not sure if this is a bug with create-vite-extra or denos npm implementation etc. I'm happy to create an issue upstream where needed....

--- a/packages/vite/src/node/config.ts
+++ b/packages/vite/src/node/config.ts
@@ -986,6 +986,7 @@ async function bundleConfigFile(
           build.onResolve({ filter: /.*/ }, ({ path: id, importer, kind }) => {
             // externalize bare imports
             if (id[0] !== '.' && !path.isAbsolute(id) && !isBuiltin(id)) {
+              // partial deno support as `npm:` does not work in `tryNodeResolve`
               if (id.startsWith('npm:')) {
                 return { external: true }
               }

--- a/packages/vite/src/node/config.ts
+++ b/packages/vite/src/node/config.ts
@@ -986,6 +986,9 @@ async function bundleConfigFile(
           build.onResolve({ filter: /.*/ }, ({ path: id, importer, kind }) => {
             // externalize bare imports
             if (id[0] !== '.' && !path.isAbsolute(id) && !isBuiltin(id)) {
+              if (id.startsWith('npm:')) {
+                return { external: true }
+              }
               let idFsPath = tryNodeResolve(id, importer, options, false)?.id
               if (idFsPath && (isESM || kind === 'dynamic-import')) {
                 idFsPath = pathToFileURL(idFsPath).href

